alacritty/src/config/mod.rs

```diff
@@ -121,6 +121,8 @@ pub fn installed_config() -> Option<PathBuf> {
                     return Some(fallback);
                 }
             }
+
+            debug!(target: LOG_TARGET_CONFIG, "No configuration file found");
```

Comment:
I feel like it should be on `info` level, since configuration loaded is also on `info` level.

Reply:
That's a great point, I just put it on the highest level that doesn't cause a UI error, but info makes more sense.